frontend/src/pages/Weather.tsx

```diff
@@ -0,0 +1,232 @@
+                                placeholder="Search location"
+                                onChange={(event) => {
+                                    console.log(event.currentTarget.value);
+                                    setCity(event.currentTarget.value);
+                                }}
+                                value={city}
+                            />
+                            <TextInput
+                                placeholder="Enter days count (1-5)"
+                                onChange={(event) => {
+                                    console.log(event.currentTarget.value);
+                                    setDays(event.currentTarget.value);
+                                }}
+                                value={days}
+                            />
+                            <Button
+                                onClick={() => {
+                                    console.log("clicked");
+                                    console.log(city);
+                                    getWeather();
+                                }}
+                                color="orange"
+                            >
+                                Get Weather
+                            </Button>
+                        </Stack>
+                    </Center>
+                    <Navbar.Section grow> </Navbar.Section>
+                    <Navbar.Section>
+                        <Group
+                            style={{
+                                display: "flex",
+                                justifyContent: "center",
+                                alignItems: "center",
+                            }}
+                        >
+                            <Button variant="default" className={classes.loginButton}>
+                                Log in
+                            </Button>
+                            <Button className={classes.signUpButton} color="orange">
+                                Sign up
+                            </Button>
+                        </Group>
+                    </Navbar.Section>
+                </Navbar>
+            }
+            footer={
+                <Footer height={40} p="md" className={classes.footer}>
+                    GG VK YB ©
+                </Footer>
+            }
+            header={
+                <Header height={{ base: 50, md: 55 }} p="md" className={classes.header}>
+                    <div
+                        style={{
+                            display: "flex",
+                            alignItems: "center",
+                            fontSize: 24,
+                            height: "100%",
+                        }}
+                    >
+                        <MediaQuery largerThan="sm" styles={{ display: "none" }}>
+                            <Burger
+                                opened={opened}
+                                onClick={() => setOpened((o) => !o)}
```

Comment:
Let's make such names more clear, for example "opened" instead of just "o"

Reply:
Fixed in 1f731021ba6750a69fc22063467b66554fa2f11a